test_scripts/WebEngine/commonWebEngine.lua

```diff
@@ -153,6 +238,107 @@ function common.processRPCSuccess(pAppId, pRPC, pData)
   mobileSession:ExpectResponse(cid, responseParams)
 end
 
+function common.createWindow(pParams, pAppId)
```

Comment:
Check if the window is necessary

Reply:
I think I now understand that some tests should ensure that widgets can be duplicated even for Web View apps.

Reply:
Yes, you are right